data/sql/updates/pending_db_world/rev_1619259290727738768.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1619259290727738768');
+
+SET @INCENDIA_AGAVE_ENTRY := 175928;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
```

Comment:
thanks for your contribution! please always use `DELETE` before `INSERT`

Reply:
@vtihomirov friendly reminder :)

Reply:
@FrancescoBorzi sorry for the delayed reply.
Two days ago I experienced a hardware failure on my dev machine and I won't be able to resolve this until I get my system up and running. Can't really give you ETA on this as I'm waiting on logistics to bring the new parts.

Cheers

Reply:
thanks for letting us know @vtihomirov and good luck with it!